presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoQueries.java

```diff
@@ -188,6 +188,10 @@ public void testSTMax()
         assertFunction("ST_YMax(ST_GeometryFromText('POLYGON ((2 0, 2 1, 3 1))'))", DOUBLE, 1.0);
         assertFunction("ST_XMax(ST_GeometryFromText('MULTIPOLYGON (((1 1, 1 3, 3 3, 3 1)), ((2 4, 2 6, 6 6, 6 4)))'))", DOUBLE, 6.0);
         assertFunction("ST_YMax(ST_GeometryFromText('MULTIPOLYGON (((1 1, 1 3, 3 3, 3 1)), ((2 4, 2 6, 6 10, 6 4)))'))", DOUBLE, 10.0);
+        assertFunction("ST_XMax(ST_GeometryFromText('POLYGON EMPTY'))", DOUBLE, null);
+        assertFunction("ST_YMax(ST_GeometryFromText('POLYGON EMPTY'))", DOUBLE, null);
```

Comment:
Can you please add dumb tests like:
```
assertFunction("ST_XMax(null)", DOUBLE, null);
assertFunction("ST_YMax(null)", DOUBLE, null);
```
You might need to us `CAST` in the above tests

Reply:
Thanks. Added these tests as well.